platform/app/package.json

```diff
@@ -87,7 +87,7 @@
     "history": "^5.3.0",
     "i18next": "^17.0.3",
     "i18next-browser-languagedetector": "^3.0.1",
-    "lodash.isequal": "4.5.0",
+    "lodash": "^4.17.21",
```

Comment:
why we are bringing the whole lodash in ohif while we just need isEqual? if we don't need the whole lodash i prefer to not have it and remove it

Reply:
I'll double check and let you know. I think I made a mistake there.

Reply:
Yes, indeed. I assumed there were other lodash functions being used because we had a "lodash" dependency in the devDependencies. But I checked and we really don't. I fixed it now. Thanks for catching that.